packages/sdk/src/global-account/react/components/SignInWithB3/steps/LoginStep.tsx

```diff
@@ -111,6 +111,7 @@ export function LoginStep({ onSuccess, onError, partnerId, chain }: LoginStepPro
           title: "Sign in with B3",
           titleIcon: "https://cdn.b3.fun/b3_logo.svg",
         }}
+        className="b3-login-step"
```

Comment:
Good call adding more classnames for customization! We should list these somewhere in docs